include/boost/ut.hpp

```diff
@@ -942,7 +942,11 @@ struct log {
 template <class TMsg>
 log(TMsg)->log<TMsg>;
 struct fatal_assertion {};
+#if defined(__cpp_exceptions)
+using exception = std::exception;
```

Comment:
Some of the builds may need a #include to go with this - or maybe the std::runtime_error below - but see comments there too...

https://travis-ci.org/boost-experimental/ut/jobs/625157540#L1534

Reply:
You're right. I thought about that later. I'll add it to the pull request